### PR TITLE
[WIPTEST] Network view fix

### PR DESCRIPTION
**Pull request by digitronik**
# Purpose or Intent
=================
- added some small fixes in network views
- tested most of with miq shell
http://pastebin.test.redhat.com/656198

- not going to cover all tests but adding one test  for @dkholodo as he only pointed me this

{{pytest: cfme/tests/networks -v}}

### CODE REVIEW

**Comment by digitronik** [comment]
part of issue: https://github.com/ManageIQ/integration_tests/issues/7960

**Comment by izapolsk** [comment]
@digitronik, please add more tests. because your changes may potentially break other network tests.

**Comment by digitronik** [comment]
@izapolsk let's run all network test against this PR so I will get idea if any test failure due to wait_for

File: cfme/networks/views.py [hunk 1]
```diff
@@ -783,6 +784,7 @@ def is_displayed(self):
 
 class FloatingIpDetailsView(BaseLoggedInPage):
     """ Represents detail view of floating ip """
+    title = Text('//div[@id="main-content"]//h1')
```
Comment (mshriver) [inline cfme/networks/views.py]: I think the title should be within entities, so these view structure are consistent.

File: cfme/networks/views.py [hunk 2]
```diff
@@ -798,3 +800,16 @@ class entities(View):  # noqa
...
+
+class OneProviderFloatingIpView(BaseLoggedInPage):
+    """ Represents Floating Ip all for specific Network provider """
+    toolbar = View.nested(OneProviderComponentsToolbar)
+    including_entities = View.include(FloatingIpEntities, use_parent=True)
+
+    @property
+    def is_displayed(self):
+        title = '{name} (All Floating IPs)'.format(name=self.context['object'].name)
+        return (super(BaseLoggedInPage, self).is_displayed and
```
Comment (mshriver) [inline cfme/networks/views.py]: Consider calling `self.logged_in_as_current_user` directly instead of the call to `super().is_displayed`

**Comment by mshriver** [comment]
@digitronik Thanks for providing some local testing results via the pastebin.

The pattern I would like to see when testing these views is:
1. Navigate and store the view
2. assert the view is displayed
3. in the browser session, navigate to any other page (good to try the other network pages)
4. assert view.is_displayed is False

In order to make sure the view doesn't falsely return True on other network pages.

**Comment by digitronik** [comment]
@mshriver I added suggestions. As per the last run, the failures were not related to changes.

File: cfme/networks/views.py [hunk 3]
```diff
@@ -735,7 +736,7 @@ def is_displayed(self):
         title = '{name} (All Security Groups)'.format(name=self.context['object'].name)
         return (super(BaseLoggedInPage, self).is_displayed and
                 self.navigation.currently_selected == ['Networks', 'Providers'] and
-                self.title.text == title)
+                self.entities.title.text == title)
```
Comment (mshriver) [inline cfme/networks/views.py]: I don't think `title` widget exists on the entities view in this case. `SecurityGroupEntities` doesn't have it, and `BaseEntitiesView` doesn't have it.

The title widget definition on L729 above needs to be moved into the `SecurityGroupEntities` view class.

Comment (digitronik) [reply cfme/networks/views.py]: No need of it. It was my bad. entities have a `title`

Comment (mshriver) [reply cfme/networks/views.py]: alright, so the title widget that was added won't be needed.

Comment (digitronik) [reply cfme/networks/views.py]: 👍 I removed Mike. It was my bad I was unknown about `BaseEntityConditionalView` even not checked with MIQ shell.

**Comment by mshriver** [comment]
PRT failures don't look related.

**Pull request merged**